lib/parsers/asm-parser.ts

```diff
@@ -615,7 +619,7 @@ export class AsmParser extends AsmRegex implements IAsmParser {
         const labelDefinitions: Record<string, number> = {};
         const dontMaskFilenames = filters.dontMaskFilenames;
 
-        let asmLines = asmResult.split('\n');
```

Comment:
took me a while to notice this was the main issue for things not to show up in binary mode on windows :(